lib/farscape/client/http_client.rb

```diff
@@ -13,7 +14,7 @@ class HTTPClient < BaseClient
 
       def initialize(agent)
         @connection = Faraday.new do |builder|
-          builder.request :url_encoded
+          builder.request :json
```

Comment:
updated to use `FaradayMiddleware::EncodeJson` for Faraday request
https://github.com/lostisland/faraday_middleware/blob/master/lib/faraday_middleware/request/encode_json.rb